### DISCUSSION TITLE
could we use this packages to calculate the DASH/MED/aMED/aHEI for NHANES 1999-2002

### DISCUSSION THREAD

**Discussion by heqsh** [jamesjiadazhan/dietaryindex#303] [Q&A] [answered]
Thank you for the package!
I was able to use it to calculate HEI-2020 and DII for NHANES 1999-2002 data, but I couldn’t calculate DASH, MED, aMED, or aHEI. Could you advise on how to resolve this issue?"

AHEI99<-AHEI_NHANES_MPED(MPED_PER_100_GRAM_PATH = NHANES_19992000$MPED_PER_100_GRAM, WJFRT = NHANES_19992000$WJFRT, NUTRIENT_PATH = NHANES_19992000$NUTRIENT, NUTRIENT_IND_PATH = NHANES_19992000$NUTRIENT_IND, DEMO_PATH = NHANES_19992000$DEMO, NUTRIENT_PATH2 = NHANES_19992000$NUTRIENT2, NUTRIENT_IND_PATH2 = NHANES_19992000$NUTRIENT_IND2)
[1] "Since no SSB code is provided, the default SSB code from 17-18 FNDDS file is used."
Error in `dplyr::mutate()`:
ℹ In argument: `ADDED_SUGAR_SSB_SERV = case_when(DR1IFDCD %in% SSB ~ ADD_SUG, TRUE ~ 0)`.
Caused by error in `case_when()`:
! Failed to evaluate the left-hand side of formula 1.
Caused by error:
! 找不到对象'DR1IFDCD'
Run `rlang::last_trace()` to see where the error occurred.

**Comment by jamesjiadazhan** [comment, marked as answer]
Thank you for this feedback. This bug is fixed.

In addition, there is no second day data for NHANES 1999-2000 and 2001-2002. Please use the first day data only for those NHANES cycles.

**Reply by heqsh** [reply]
Thanks, I'll try again.
